artiq/master/scheduler.py

```diff
@@ -122,6 +122,13 @@ def __init__(self, ridc, worker_handlers, notifier, experiment_db):
         self.notifier = notifier
         self.experiment_db = experiment_db
 
+    def log_tuples(self, rid, expid):
```

Comment:
"Tuples" is not a good name in functions either. What about ``log_submissions``?